app/src/main/res/values/strings.xml

```diff
@@ -194,22 +195,22 @@
     <string name="share_title">Share</string>
     <string name="share_menu_send">Share</string>
     <string name="share_content_title">Message preview</string>
-    <string name="share_content_text_hint">Add the content you\'d like to share here</string>
+    <string name="share_content_text_hint">Add content to share here</string>
     <string name="share_content_image_text">An image was shared with you</string>
     <string name="share_content_image_error">Cannot read image: %1$s</string>
     <string name="share_content_file_text">A file was shared with you</string>
     <string name="share_content_file_error">Cannot read file infos: %1$s</string>
     <string name="share_topic_title">Share to</string>
     <string name="share_suggested_topics">Suggested topics</string>
-    <string name="share_successful">Message successfully published</string>
+    <string name="share_successful">Message published</string>
 
     <!-- Notification dialog -->
     <string name="notification_dialog_title">Pause notifications</string>
     <string name="notification_dialog_cancel">Cancel</string>
     <string name="notification_dialog_save">Save</string>
     <string name="notification_dialog_enabled_toast_message">Notifications re-enabled</string>
     <string name="notification_dialog_muted_forever_toast_message">Notifications paused</string>
-    <string name="notification_dialog_muted_until_toast_message">Notifications paused until %1$s</string>
+    <string name="notification_dialog_muted_until_toast_message">Notifications paused till %1$s</string>
```

Comment:
I like "until" much more. Isn't "till" colloquial?

Reply:
Interchangeable https://grammar.collinsdictionary.com/english-usage/what-is-the-difference-between-until-and-till